From: Michael Froman <mjfroman@mac.com>
Date: Tue, 17 Sep 2024 13:55:10 -0500
Subject: (tmp-cherry-pick) Revert "Reland "Add a dependancy on
 //third_party/jni_zero:jni_zero_java"" (d02a216026)

This reverts commit e19a9b37ce477f867524dfb802cca556630a7fac.

Reason for revert: break downstream

Original change's description:
> Reland "Add a dependancy on //third_party/jni_zero:jni_zero_java"
>
> This is a reland of commit 5ccec98826adc84bdb4c8177580a6f23074d8058
>
> Original change's description:
> > Add a dependancy on //third_party/jni_zero:jni_zero_java
> >
> > JniZero's native side expects JniZero's java side to exist, add a
> > dependancy on //third_party/jni_zero:jni_zero_java from
> > sdk/android:base_java.
> >
> > Bug: webrtc:353174456
> > Change-Id: I9240eab3da78efd394e4ee581cf71fd41976eb48
> > Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/357200
> > Commit-Queue: Mohamed Heikal <mheikal@google.com>
> > Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
> > Auto-Submit: Mohamed Heikal <mheikal@google.com>
> > Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
> > Cr-Commit-Position: refs/heads/main@{#42639}
>
> Bug: webrtc:353174456, b/325408567
> Change-Id: Ib51939cc6494e867e8018c7d163c85f36b80199b
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/358260
> Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
> Reviewed-by: Jeremy Leconte <jleconte@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#42709}

Bug: webrtc:353174456, b/325408567
Change-Id: I2eb497d8a308485142f6e197e8e41928e2ae6f25
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/358280
Auto-Submit: Jeremy Leconte <jleconte@google.com>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Commit-Queue: Jeremy Leconte <jleconte@google.com>
Owners-Override: Jeremy Leconte <jleconte@google.com>
Cr-Commit-Position: refs/heads/main@{#42710}
---
 sdk/android/BUILD.gn | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sdk/android/BUILD.gn b/sdk/android/BUILD.gn
index 641444b301..be84a6158b 100644
--- a/sdk/android/BUILD.gn
+++ b/sdk/android/BUILD.gn
@@ -172,7 +172,6 @@ if (is_android) {
     deps = [
       "//rtc_base:base_java",
       "//third_party/androidx:androidx_annotation_annotation_java",
-      "//third_party/jni_zero:jni_zero_java",
     ]
   }
 
